e2e/packages/sync-test/package.json

```diff
@@ -6,6 +6,9 @@
   "scripts": {
     "test": "vitest --run"
   },
+  "dependencies": {
+    "zod": "^3.22.2"
```

Comment:
nit: i think this could be a dev dependency?

Reply:
oh sure, I guess the whole package is ~dev so I wasn't paying too close attention to it

Reply:
yeah it's not important, just OCD inducing that one dependency is not marked as dev